### PR TITLE
mk-notes: Allow 'under development' version

### LINKED ISSUE #46: Section Formatting Issue - Wrong Version?
Something is wrong with formatting here: https://www.openssl.org/news/openssl-1.1.1-notes.inc that is affecting the HTML version: https://www.openssl.org/news/openssl-1.1.1-notes.html
The text version has the updates correctly appearing in the 1.1.0 notes file: https://www.openssl.org/news/cl110.txt

I'm not sure how to submit a pull request for this problem because the source files do not appear to be in this repository.

### PR DESCRIPTION
**Pull request by jrchamp**
Fixes #46

### CODE REVIEW

File: bin/mk-notes [hunk 1]
```diff
@@ -8,7 +8,7 @@ my $copy = 0;
 my $in_ul = 0;
 while ( <STDIN> ) {
     chomp;
-    if (/^\s*(Major changes between|Known issues in).*(\d+\.\d+\.\d+)\D.*\[(in pre-release|\d+\s(Jan|Feb|Mar|Apr|May|Jun|Jul|Aug|Sep|Oct|Nov|Dec)\s\d+)\]:?$/) {
+    if (/^\s*(Major changes between|Known issues in).*(\d+\.\d+\.\d+)\D.*\[(in pre-release|under development|\d+\s(Jan|Feb|Mar|Apr|May|Jun|Jul|Aug|Sep|Oct|Nov|Dec)\s\d+)\]:?$/) {
```
Comment (levitte) [inline bin/mk-notes]: To get the a result that does corresponds to what NEWS actually says, I believe you need to insert the following below this line:

``` perl
        next unless $4;
```

**Review by levitte — CHANGES_REQUESTED** [review]

File: bin/mk-notes (hunk 1 above)
Comment (levitte) [reply bin/mk-notes]: That should mean it will stop the process only at *dated* titles.

Comment (jrchamp) [reply bin/mk-notes]: ``next unless $4;`` will prevent the 1.1.1 "in pre-release" notes from being generated. I have another solution if you still want the "in pre-release" notes.

**Comment by jrchamp** [comment]
With the latest commit, it will always identify a header line. Then, the release "date" is checked to see if the section (both header and content) should be printed. Finally, the close ``ul`` is only printed if we printed the open tag.

**Comment by richsalz** [comment]
@levitte just a reminder (and a public statement for those following this thread) we don't have a review policy on the web repo.  If you like this, or want to make changes and merge, go ahead.

**Comment by levitte** [comment]
Did merge it, and added a small mod.

Thanks for your contribution, @jrchamp

**Pull request closed without merging**